index.css

```diff
@@ -17,14 +17,15 @@ html {
  * GENERAL STUFF
  */
 .btn {
+  display: inline-block;
   -webkit-appearance: none;
   background: #f0f0f0;
   border: none;
   border-radius: 4px;
   color: #000;
   cursor: pointer;
   margin: 0;
-  padding: 0.7em 1em;
+  padding: 0.5em 1em;
```

Comment:
Do we want to use `rem` instead of `em`? The main difference is that `rem` is relative to the root HTML tag rather than `em`, which is relative to the container.

https://j.eremy.net/confused-about-rem-and-em/

Reply:
yeh it was initially rem but I wanted the button to get bigger as the text size increased.

Reply:
Noted